validation/gwas/method/pc_relate/README.md

```diff
@@ -0,0 +1,19 @@
+This runs test to validate our implementation gets the same
+exact results as the reference R implementation for production
+HapMap data.
+
+This code is scheduled as part of the Github Actions CI.
+
+To run manually:
+
+```bash
+gsutil -u $BILLING_PROJECT cp gs://sgkit-data/validation/hapmap_JPT_CHB_r23a_filtered.zip /tmp/
+./run.sh /tmp/hapmap_JPT_CHB_r23a_filtered.zip
+```
```

Comment:
Might be worth adding a comment about requester-pays and the data size to the README.

Reply:
Fixed